cluster-provision/okd/scripts/run.sh

```diff
@@ -76,7 +91,12 @@ until [[ $(oc -n openshift-ingress get pods -o custom-columns=NAME:.metadata.nam
 done
 
 # update hostnames for services to point to the node with the route pod
-worker_node_ip=$(oc -n openshift-ingress get pods -o custom-columns=NAME:.metadata.name,HOST_IP:.status.hostIP,PHASE:.status.phase | grep route | grep Running | head -n 1 | awk '{print $2}')
+worker_node_ip=""
+until [[ ${worker_node_ip} != "" ]]; do
```

Comment:
This pattern of retrying to get something repeats a lot. Could we make a helper function with timeout for it?